src/pca_report_library/customer/closing.py

```diff
@@ -16,8 +16,8 @@
 from itertools import tee
 import json
 
-# cisagov Libraries
-from utility.gets import get_level_description, get_max_number
+# from pca_report_library.utility.gets import get_level_description, get_max_number
```

Comment:
This commented-out line should be deleted:
```suggestion
```

So should this other redundant line:
https://github.com/cisagov/pca-report-library/blob/a857743d5a47b63050241a34b6f0d30526e58984/src/pca_report_library/customer/closing.py#L15

Reply:
Resolved in 495907b and 495907b